### ISSUE TITLE
Input file formatting (PDB/PDBQT)

### ISSUE THREAD

**Issue by mscockatiel** [RedesignScience/AlphaSpace2#19]
Hello,
I am currently having issues conducting experimentation using the scripts you provide in the tutorial. Specifically, the tutorial 4 "fragment to lead" is not working. I didn't change the tutorial script other than the input file names and locations. I believe that the issue is regarding my input files, as it runs without error when using the files given in the tutorial. I have compared the tutorial files and my files, and can't understand what is missing. I made sure that the spacing within the files were the same. The error I get is included below. I could email you the pdb and pdbqt files I am trying to use as well if that would be helpful. 
 
Thank you in advance!
<img width="716" alt="error message" src="https://user-images.githubusercontent.com/71343207/99590995-24079800-29bc-11eb-9dec-946b7239222d.png">

**Comment by lenhsherr** [comment]
Can you attach the pdbqt file in the issues? Also, you need to remove the hydrogen for both of pdb and pdbqt file to work.

**Comment by mscockatiel** [comment]
Yes, I believe that there are no hydrogens in either of my files.
